best-practices/development-tools/azure/devops/README.md

```diff
@@ -0,0 +1,12 @@
+# Deploying a .NET CORE APP using Azure DevOps
+
+## Deploying into AWS Elastic Beanstalk
+- [Elastic Beanstalk Deployment](deployment/aws-elasticbeanstalk/aws-elastic-beanstalk-deployment.md)
```

Comment:
Since most people will be coming into this blind, how do you feel about adding a small description under AWS Elastic Beanstalk and Azure APP Functions on what they do?

Reply:
Good suggestion.